src/rad/resources/manifests/datamodels-1.0.yaml

```
@@ -251,6 +251,11 @@ tags:
   title: Skycells Reference File Schema
   description: |-
     This file contains definitions for all the skycells that cover the entire celestial sphere
+- tag_uri: asdf://stsci.edu/datamodels/roman/tags/reference_files/matable-1.0.0
+  schema_uri: asdf://stsci.edu/datamodels/roman/schemas/reference_files/matable-1.0.0
+  title: MA Table Reference File Schema
```

```suggestion
  title: Multiple Accumulation Table Reference File Schema
```
To match the title in the schema.

---

Changed.